gui/src/components/maintenance/cortx-auditlog.vue

```diff
@@ -15,7 +15,7 @@
 * please email opensource@seagate.com or cortx-questions@seagate.com.
 */
 <template>
-  <div v-feature="unsupportedFeatures.auditlog">
```

Comment:
Why is this removed? This should not be removed. The decision will be in backend whether auditlog feature is supported or not.